pkg/operator/ceph/cluster/mon/mon.go

```diff
@@ -82,6 +83,11 @@ const (
 	// default storage request size for ceph monitor pvc
 	// https://docs.ceph.com/docs/master/start/hardware-recommendations/#monitors-and-managers-ceph-mon-and-ceph-mgr
 	cephMonDefaultStorageRequest = "10Gi"
+
+	// canary pod scheduling uses retry loops when cleaning up previous canary
+	// pods and waiting for kubernetes scheduling to complete.
+	canaryRetries           = 15
+	canaryRetryDelaySeconds = 60
```

Comment:
The operator should be fairly responsive when the canary pod is ready. Since it's cheap to check if the canary pod is ready, what if `canaryRetryDelaySeconds` were 5 seconds? The retry number will just be a lot bigger.